features/step_definitions/ocm.rb

```diff
@@ -1,3 +1,4 @@
+require 'webauto/webconsole_executor'
```

Comment:
we shouldn't need this line

Reply:
@akostadinov I met some error as bellow, and it will be fixed by adding this line, That's just as a workaround. The error I met is 
`Given I open ocm portal as an orgAdmin user                                      # features/step_definitions/ocm/ocm.rb:2
      uninitialized constant BushSlicer::WebConsoleExecutor (NameError)
    /root/ocm_ui_automation_place_3/verification-tests/features/step_definitions/ocm/ocm.rb:3:in `/^I open ocm portal as an? (\w+|<%=.+?%>) user$/'
      features/tierN/ocm/cluster_actions.feature:6:in `Given I open ocm portal as an orgAdmin user'
`